lang_qc/models/pacbio/experiment.py

```diff
@@ -19,14 +19,96 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <http://www.gnu.org/licenses/>.
 
-from typing import List
+from typing import Any
 
-from pydantic import BaseModel, ConfigDict, Field
+from pydantic import Field, model_validator
+from pydantic.dataclasses import dataclass
 
 from lang_qc.db.mlwh_schema import PacBioRun
 
 
-class PacBioExperiment(BaseModel):
+@dataclass(kw_only=True, frozen=True)
+class PacBioLibrary:
```

Comment:
This isn't a million miles away from being just `Library`, subject to warehouse table resolution.

Reply:
True. If we need another one like this for a different platform, the fields can go to lang_qc::models::library and the build method stays in this class or something like this. I had it like this, then decided against it for now